test/src/test/java/test/compatability/BridgeMethodsTest.java

```diff
@@ -0,0 +1,36 @@
+package test.compatability;
+
+import java.lang.reflect.Method;
+import java.util.Arrays;
+import java.util.List;
+import java.util.concurrent.Future;
+import java.util.stream.Collectors;
+import org.junit.Test;
+import org.jvnet.hudson.test.Issue;
+import hudson.model.Queue;
+import hudson.model.queue.QueueTaskFuture;
+import static org.hamcrest.CoreMatchers.allOf;
+import static org.hamcrest.CoreMatchers.hasItem;
+import static org.hamcrest.CoreMatchers.is;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.beans.HasPropertyWithValue.hasProperty;
+import static org.hamcrest.collection.IsIterableWithSize.iterableWithSize;
+
+public class BridgeMethodsTest {
+
+    @Test
+    @Issue("[JENKINS-65605")
```

Comment:
Extra [

Reply:
fixed in b36eec37598edfd20dc063e5380421befe1c03f0